ansys/mapdl/core/commands.py

```diff
@@ -211,4 +211,34 @@ class Commands(
 
     """Wrapped MAPDL commands"""
 
-    pass
+
+class CommandOutput(str):
+    """Customized Command Output class"""
+
+    ## References:
+    # - https://stackoverflow.com/questions/7255655/how-to-subclass-str-in-python
+    # - https://docs.python.org/3/library/collections.html#userstring-objects
+    # - Source code of UserString
+
+    def __new__(cls, content, cmd=None):
+        obj = super().__new__(cls, content)
+        obj._cmd = cmd
+        return obj
```

Comment:
Ha! You monkey patched `str`. Good approach.

Reply:
I felt it does the job, while keeping most of the design constrains. My biggest problem was to i"nheritate" the `_cmd` property actually.